site/templates/origin__3_harryette_mullen.php

```diff
@@ -1,7 +1,64 @@
-<?php snippet('header'); ?>
+<?php snippet('header', ['headerClass' => 'mullen']); ?>
+<div class="navigation__container">
+  <nav>
+    <ul class="navigation">
+      <li class="navigation__item highlight"><a href="<?= $site->url(); ?>"><?= $site->title(); ?></a></li>
+    </ul>
+  </nav>
+</div>
+
 <div class="layout-wrapper">
-  <div class="text">
-    <?= $page->main_content()->kt(); ?>
+
+
+    <div class="poem">
+    <?= $page->poem()->kt(); ?>
+    </div>
+
+
+    <div class="poem_citation">
+    <?= $page->poem_citation()->kt(); ?>
+    </div>
+
+    <!-- <?php foreach($page->blocks()->toStructure() as $block): ?>
+
+    <div class="audio__caption">
+      <?= $block->block_text()->kt(); ?>
+    </div>
+    <?php endforeach; ?> -->
+
+<ul>
+  <div class="audio">
+
+      <?php foreach($page->blocks()->toStructure() as $block): ?>
+
+        <div class="audio__caption">
```

Comment:
if using `kt()` in php, add a class `text`, which is needed to any kirby text properly. 

should read `audio__caption text`

Reply:
lets go over this adding text for kirby im a bit unclear why i will add text to code tho